backend/muster-roll-service/src/main/java/org/egov/web/controllers/MusterRollApiController.java

```diff
@@ -0,0 +1,81 @@
+package org.egov.web.controllers;
+
+
+import java.math.BigDecimal;
+
+import digit.models.coremodels.RequestInfoWrapper;
+import org.egov.web.models.MusterRollRequest;
+import org.egov.web.models.MusterRollResponse;
+import org.egov.common.contract.request.RequestInfo;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.swagger.annotations.*;
+import org.egov.web.models.MusterRollSearchCriteria;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.io.IOException;
+
+import javax.validation.constraints.*;
+    import javax.validation.Valid;
+    import javax.servlet.http.HttpServletRequest;
+
+@javax.annotation.Generated(value = "org.egov.codegen.SpringBootCodegen", date = "2022-11-14T19:58:09.415+05:30")
+
+@Controller
+    @RequestMapping("/v1")

```

Comment:
Please run Reformat Code on all files to adjust tab spaces.

Reply:
Applied refactor code on all files